packages/app/src/pages/PoolPage/AddLiquidity.tsx

```diff
@@ -282,6 +283,17 @@ export default function AddLiquidity() {
           ? "Add liquidity"
           : "Create liquidity"}
       </Button>
+      {!reservesFromToRatio ? (
```

Comment:
During the loading of the pool_info, the page is showing maybe check if is not loaded yet

Reply:
nice catch